src/components/PyatchAddSprite.jsx

```diff
@@ -70,6 +71,7 @@ export function PyatchAddSprite(props) {
                 <MenuItem key="new" onClick={handleUploadNew}>Upload new costume</MenuItem>
             </Menu>
             <PatchInternalSpriteChooser pyatchVM={pyatchVM}/>
+            <PatchInternalSoundChooser pyatchVM={pyatchVM}/>
```

Comment:
Don't need to pass pyatchVM here as these elements should grab it from useContext